lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -494,7 +495,7 @@ expectWalletUTxO coins = \case
         let addr = Address "ARBITRARY"
         let constructUtxoEntry idx c =
                 ( TxIn (Hash "ARBITRARY") idx
-                , TxOut addr (Coin c)
+                , TxOut addr (TB.fromCoin $ Coin c)
```

Comment:
> Minor suggestion because TB feels rather cryptic first reading it; what about Bundle? Bundle.fromCoin reads without ambiguity.

@KtorZ I also like your suggestion of `Bundle`, but it raises the question of how to other modules like `TokenMap` in a qualified way.

The PR currently import token-related modules in the following standard ways:
```
import qualified Primitive.Types.TokenBundle   as TB
import qualified Primitive.Types.TokenMap      as TM
import qualified Primitive.Types.TokenQuantity as TQ
```
We could make this a bit more verbose:
```
import qualified Primitive.Types.TokenBundle   as TBundle
import qualified Primitive.Types.TokenMap      as TMap
import qualified Primitive.Types.TokenQuantity as TQuantity
```
Or even just use the whole name:
```
import qualified Primitive.Types.TokenBundle   as TokenBundle
import qualified Primitive.Types.TokenMap      as TokenMap
import qualified Primitive.Types.TokenQuantity as TokenQuantity
```

Would any of these options work for you?

Reply:
I'd actually merge those modules together as `import qualified Primitive.Types.TokenBundle` and import it qualified as `Bundle` :no_mouth: 

The modules are very closely related and actually sharing data-structures from one another. A change in `TokenQuantity` will likely induce a change in `TokenMap` and then `TokenBundle`, so it makes sense to keep them close. Although it may not play well with the modules API which maybe overlap a bit. 

From the choices you've pointed out, the last one (whole name) is what reads clearer in my opinion. And in the end roughly the same to type. `To<tab>` will do in most cases :)

Reply:
> `import qualified Primitive.Types.TokenBundle   as TokenBundle`
> `import qualified Primitive.Types.TokenMap      as TokenMap`
> `import qualified Primitive.Types.TokenQuantity as TokenQuantity`

@KtorZ wrote:
> From the choices you've pointed out, the last one (whole name) is what reads clearer in my opinion. And in the end roughly the same to type. `To<tab>` will do in most cases :)

Okay!

I've fixed this in ca948b4ed9046e9e6030c29a1f7055c596aab0e7.